migrations/20171130101357_add_first_admin_to_tos.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AddFirstAdminToTos extends AbstractMigration
+{
+   /**
+     * Migrate Up.
+     */
+    public function up()
+    {
+        $pdo = $this->getAdapter()->getConnection();
+        $user = $pdo->query("SELECT * FROM users where id = 1")->fetch();
```

Comment:
On ushahidi.io is the first admin reliably id 1? 

Another approach to this might be to add a `--tos` flag to the user create CLI tasks. That task is used by the cloud job queue to create the admin user on new deployments.

Reply:
Good point, I was thinking it is the same as the admin created in the migrations but of course it is not... :/